template/webpack.config.js

```diff
@@ -70,10 +111,7 @@ const config = (platform, launchArgs) => {
           test: /\.vue$/,
           loader: 'ns-vue-loader',
           options: {
-            loaders: {
-              css: cssLoader,
-              scss: scssLoader,
-            },
+            extractCSS: true,
```

Comment:
Ideally we could set this to false when in debug mode, but it seems to break `nativescript-vue` somehow.